frontend/src/Activity/Blocklist/Blocklist.js

```diff
@@ -215,6 +230,16 @@ class Blocklist extends Component {
           onConfirm={this.onRemoveSelectedConfirmed}
           onCancel={this.onConfirmRemoveModalClose}
         />
+
+        <ConfirmModal
+          isOpen={isConfirmClearModalOpen}
+          kind={kinds.DANGER}
+          title={translate('Clear')}
```

Comment:
```suggestion
          title={translate('ClearBlocklist')}
```

(along with an associated translation added)